device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/edge/MethodRequest.java

```diff
@@ -34,15 +34,15 @@
     private static final String PAYLOAD_KEY_NAME = "payload";
     @Expose(deserialize = false)
     @SerializedName(PAYLOAD_KEY_NAME)
-    private String payload;
+    private Object payload;
 
     /**
```

Comment:
Can you rename this class from MethodRequest to DirectMethodRequest for naming consistency?